test/hard_swish_test.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+import {hardSwish} from '../src/hard_swish.js';
+import {Tensor} from '../src/lib/tensor.js';
+import * as utils from './utils.js';
+
+describe('test hardSwish', function() {
+  function testHardSwish(input, expected) {
+    const inputTensor = new Tensor(input.shape, input.value);
+    const outputTensor = hardSwish(inputTensor);
+    utils.checkValue(outputTensor, expected);
+  }
+
+  // this input data referres to NNAPI CTS tests of HARD_SWISH op
+  //   https://android.googlesource.com/platform/frameworks/ml/+ \
+  //       /master/nn/runtime/test/specs/V1_3/hard_swish.mod.py#50
+  const inputData = [
+    4.53125, 3.90625, 3.046875, -8.59375, -1.328125,
+    1.328125, 0.0, -8.515625, -8.984375, -0.234375,
+    0.859375, 9.84375, -0.15625, -8.515625, 8.671875,
+    4.609375, 9.21875, -1.796875, 1.171875, 9.375,
+    -8.75, 2.421875, -8.125, -1.09375, -9.609375,
+    -1.015625, -9.84375, 2.578125, 4.921875, -5.078125,
+    5.0, -0.859375, 1.953125, -6.640625, -7.8125,
+    4.453125, -4.453125, -6.875, 0.78125, 0.859375,
+  ];
+  const expectedData = [
+    4.53125, 3.90625, 3.046875, -0.0, -0.3700764973958333,
```

Comment:
Interesting that `math.Max(0, -8)` returns -0 instead of 0, but it should make no difference either way. (no change requested)